src/bp/ui-studio/src/web/views/FlowBuilder/diagram/manager.ts

```diff
@@ -40,6 +42,10 @@ const createNodeModel = (node, modelProps) => {
     return new ListenNodeModel(modelProps)
   } else if (type === 'router') {
     return new RouterNodeModel(modelProps)
+  } else if (type === 'comment') {
+    const node = new CommentNodeModel(modelProps)
+    node.handleTextUpdate = engine.flowBuilder.updateCommentText
```

Comment:
Is there a better way to call `Diagram`.`updateCommentText()` from `CommentNodeModel`?